lib/src/firebase_auth_mocks_base.dart

```diff
@@ -58,6 +58,6 @@ class MockFirebaseAuth extends Mock implements FirebaseAuth {
   }
 
   @override
-  Stream<User> get onAuthStateChanged =>
```

Comment:
Since onAuthStateChanged has been deprecated but not removed yet, it'd have been better to keep the onAuthStateChanged getter and make it call authStateChanges(). But for now, I'll merge this. Thanks!

Reply:
I ended up restoring the deprecated getter, and published your PR at https://pub.dev/packages/firebase_auth_mocks/changelog#032. Thanks again @gallrein